packages/cli/scaffold/resource.template.ts

```diff
@@ -0,0 +1,119 @@
+import type { PerformFunction, Resource } from 'zapier-platform-core';
+
+// get a list of <%= LOWER_NOUN %>s
+const performList: PerformFunction = async (z, bundle) => {
+  const response = await z.request({
+    url: 'https://jsonplaceholder.typicode.com/posts',
+    params: {
+      order_by: 'id desc',
+    },
+  });
+  return response.data;
+};
+
+// find a particular <%= LOWER_NOUN %> by name (or other search criteria)
+const performSearch: PerformFunction = async (z, bundle) => {
+  const response = await z.request({
+    url: 'https://jsonplaceholder.typicode.com/posts',
+    params: {
+      name: bundle.inputData.name,
+    },
+  });
+  return response.data;
+};
+
+// creates a new <%= LOWER_NOUN %>
+const performCreate: PerformFunction = async (z, bundle) => {
+  const response = await z.request({
+    method: 'POST',
+    url: 'https://jsonplaceholder.typicode.com/posts',
+    body: {
+      name: bundle.inputData.name, // json by default
+    },
+  });
+  return response.data;
+};
+
+export default {
+  // see here for a full list of available properties:
+  // https://github.com/zapier/zapier-platform/blob/main/packages/schema/docs/build/schema.md#resourceschema
+  key: '<%= KEY %>',
+  noun: '<%= NOUN %>',
+
+  <%= INCLUDE_INTRO_COMMENTS ? [
+    '// If `get` is defined, it will be called after a `search` or `create`'
+  ].join('\n    ') : '' %>
+  // useful if your `searches` and `creates` return sparse objects
+  // get: {
+  //   display: {
+  //     label: 'Get <%= NOUN %>',
+  //     description: 'Gets a <%= LOWER_NOUN %>.'
+  //   },
+  //   operation: {
+  //     inputFields: [
+  //       {key: 'id', required: true}
+  //     ],
+  //     perform: defineMe
+  //   }
+  // },
```

Comment:
You want to uncomment this?

Reply:
This is standard in the [JS Resource template](https://github.com/zapier/zapier-platform/blob/e5a3ca1d1254910037e9041e2b568923f2234515/packages/cli/scaffold/resource.template.js#L41-L56), generated when `--no-help` is not provided for extra docs.